core/src/main/kotlin/in/specmatic/core/Feature.kt

```diff
@@ -400,6 +394,27 @@ data class Feature(
         serverState = emptyMap()
     }
 
+    private fun addIndexes(
```

Comment:
Technically this method is adding both `+v`/`-ve` prefix as well as index, should it be named appropriately?

Reply:
Will also be super helpful to unit test this method separately. While the feature level tests are very useful, this will allow is try all difficult combinations.

Reply:
Is this method doing too much, should we separate out application of prefix and index as two separate `collections.map` operations?

Reply:
###  ToDo's
- [x] Technically this method adds both the +v/-ve prefix and the index, should it be named appropriately?
- [ ] It will also be super helpful to unit test this method separately. While the feature-level tests are very useful, this will allow us to try all difficult combinations.
- [ ] Is this method doing too much, should we separate the application of prefix and index as two separate collections.map operations?